examples/ntzdebruijn/src/main.leo

```diff
@@ -0,0 +1,53 @@
+// The 'ntzdebruijn' main function.
+// From Hacker's Delight 2nd ed. figure 5-26
```

Comment:
Can we include a link to this algorithm in the `README.md`?

Reply:
Added info in the `README.md` files.